locale/en/download/package-manager.md

```diff
@@ -237,6 +238,47 @@ from source:
 nvm uninstall 8
 ```
 
+## nvs
+`nvs` is a version manager for Windows.
+
+The `nvs` version manager is actually cross-platform so can be used on  Unix / OS X systems as well but `nvm` is a lot more popular there.
```

Comment:
"actually" implies others aren't; `nvm` works on WSL, and also now cygwin, and git bash, and Powershell.

Reply:
Thanks for the feedback. I have added that.